devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy

```diff
@@ -225,6 +234,19 @@ def refresh() {
 	delayBetween(commands,100)
 }
 
+def healthPoll() {
+	def cmds = refresh()
+	cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
+}
+
+def configure() {
+	unschedule()
+	runEvery5Minutes(healthPoll)
```

Comment:
I agree that Z-Wave dimmers are being polled as needed from the hub, so shouldn't be polled from the cloud regularly, that's why I think of using a simplistic approach towards ping().
Hence, removed the scheduling of healthPoll().